### ISSUE TITLE
[A11Y] Close control missing accessible name

### ISSUE THREAD

**Issue by PhilibertDugas** [paypal/paypal-checkout#784]
The PayPal overlay Close control is missing an accessible name in order to convey its purpose. This would cause a poor or confusing user experience for screen reader users. Users who have mobility-related disabilities using voice recognition software will have difficulty accessing this control.

**HTML:**
```html
<a href="#" class="paypal-checkout-close"></a>
```
## Steps to reproduce

1. Go to https://www.chubbiesshorts.com/ with any supported browser
2. Add some items to the cart
3. Proceed to the Cart
4. Activate the PayPal control
5. `Cmd` + `~` back to the main window
6. With an active screen reader, `Tab` to the Close control

### Results

**Expected:**
- Controls to have text (accessible name) which describes its purpose.

**Actual:**
- Control has no text to describe its purpose.

## Recommendation

- Add the `aria-label` attribute with an appropriate value to provide context for the control.

**HTML:**
```html
<a href="#" class="paypal-checkout-close" aria-label="Close dialog"></a>
```
## Additional information

- **WCAG Principle:** Operable
- **WCAG SC:** 2.4.4 Link Purpose
- **Severity:** High

## Screenshots
<img width="702" alt="screen shot 2018-03-05 at 2 22 31 pm" src="https://user-images.githubusercontent.com/1392632/36994943-b3d55050-2080-11e8-8375-6905fff0c56f.png">

**Comment by bluepnume** [comment]
@vishakha94 can you take this one?

**Comment by mcntsh** [comment]
It looks like there was an accessibility fix for this button back in March: https://github.com/paypal/paypal-checkout/commit/bddf347b6
